CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## On the `main` branch
 
+### Fixes
+
+- `BUILD_URI` for GitHub actions now includes run attempt
+  in the URI. Previously `BUILD_URI` always linked to
+  latest attempt.
```

Comment:
as English seems to be your strong suit, can we do one pass before each release rewording changelog? otherwise it just slows down PR process as it requires:

* adding more commits to PR
* which reruns test suite
* requires follow up approval as new diff will invalidate any existing approvals?